app/modules/ads/setup.js

```diff
@@ -54,6 +55,7 @@ export const adsSetup = {
     templateService.register(BigFancyAdAbove, getBfaaConfig());
     templateService.register(BigFancyAdBelow, getBfabConfig());
     templateService.register(FloorAdhesion, getOutOfPageConfig());
+    templateService.register(HideOnViewability, {});
```

Comment:
Second argument of `templateService.register` is optional, why pass empty object?

Reply:
Yep, removed.